packages/tasks/src/tasks/zero-shot-object-detection/about.md

```diff
@@ -0,0 +1,7 @@
+## About the Task
```

Comment:
This page is already contributed in another PR, so if you feel like contributing it, you can open PR to that branch as that one is more extensive